qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddEditAppointmentBlockTest.java

```diff
@@ -46,8 +46,9 @@ public void addEditAppointmentBlockTest() throws Exception {
         appointmentBlocksPage.selectLocation(locationName);
         appointmentBlocksPage.clickOnDay();
         appointmentBlocksPage.enterStartTime(startTimeFirstAppointment);
-        appointmentBlocksPage.enterService("gyne");
```

Comment:
What happens when this is present.

Reply:
"gyne" is an acronym to mean Gynecology as per the test definition. So the search indexing criteria some times is able to map the acronym to the Service type and some times fails. 

I also noticed from the UI that one has to enter Provider first before entering the service type.

Reply:
According to your explanation above. it looks fairly enough to be approved